playbooks/vars/guests.yml

```diff
@@ -17,7 +17,7 @@ guests:
   network: bridge=virbr0
   os_type: linux
   os_variant: rhel7.6
-  disk_os_size: 40g
+  disk_os_size: 500g
```

Comment:
straight from 40 to 500? What are you putting on the OS disk?

Reply:
... The sort answer is: id did not work work with just 40g.
and the big structural change is starting glusterfs on ( master, infra, node ).
and the  mount /dev/vdb is reserved for glusterfs -- so i **assume** that the System and glusterfs share the FS on each node.
did Request advice per mail.